mobile_verifier/src/heartbeats.rs

```diff
@@ -94,167 +146,248 @@ impl HeartbeatDaemon {
         .await
     }
 
-    async fn process_file(
+    async fn process_cell_hb_file(
```

Comment:
we could add a generic struct to the filestore but then we loose granularity.  Whilst the wifi HBs and the cbrs HBs end up cast to a generic common struct I think that struct is better homed in the oracle itself rather than baked into the filestore

Reply:
not sure i understand the case for "granularity" when the only difference in how they're being handled is in the trace that's being emitted. i'm suggesting making the second argument generic as in `FileInfoStream<T>` where `T` satisfies the requirement that the two reports satisfy they define a variant of the heartbeat ingest report.

i know this implementation has changed to have two separate files that differentiate processing wifi vs. cbrs ingest reports but, in keeping with @maplant 's suggestion of a `heartbeats` directory with a nested `cbrs.rs` and `wifi.rs` module inside and the shared functionality in the `mod.rs`, this generic implementation of the `process_file` function would go there.